backend/src/ploneorg/src/ploneorg/content/plonerelease.py

```diff
@@ -82,6 +86,11 @@ def title(self):
     def Title(self):
         return self.title
 
+    def setTitle(self, value):
+        # Volto tries to set the title to None.  We ignore this.
+        # See https://github.com/plone/plone.org/issues/46
+        logger.warning("Ignoring setTitle on PloneRelease. Requested value: %r", value)
```

Comment:
@mauritsvanrees I would do it like this so that it is also used when `obj.title = x` is called directly:

```
@title.setter
def title(self, value):
    pass
```

(I wouldn't bother logging a warning; I've used this approach for other content types that have an automatically generated title. Warnings are for abnormal behavior but avoiding setting the title is normal here.)

Reply:
Good points. Let me combine those: only warn when the value is not empty. Expected is that the title is None.

A normal request is this:

```
curl 'http://admin:admin@localhost:8080/Plone/++api++/download' -X 'POST' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"release_date":"2022-12-05","release_notes":{"data":"<p> </p>"},"version":"6.0.0rc2","@static_behaviors":null,"@type":"plonerelease"}'
```

Fix committed.